tests/e2e/cypress/integration/Resources-status/resourceCommon.ts

```diff
@@ -0,0 +1,19 @@
+const containerStateFilter = 'div[aria-label="State filter"]';
+const defaultfilterValue = 'unhandled_problems';
+const labelClearFilter = 'Clear';
+
+const selectFilterOnInput = (filterValue = defaultfilterValue) => {
+  const inputFilter = 'input[aria-label="MuiSelect-nativeInput"]';
+
+  cy.get(inputFilter).should('not.be.empty');
+
+  cy.get(containerStateFilter).should('be.visible').click();
+  cy.get(`li[data-value="${filterValue}"]`).click();
```

Comment:
Same here, as we click on the filter first, we get a list of texts representing their associated filter (See the Filter/index.test.tsx of the Resource page)